### ISSUE TITLE
New UI compatability

### ISSUE THREAD

**Issue by keriefie** [blai30/Lotus-Theme#10]
![image](https://github.com/blai30/Lotus-Theme/assets/124537276/b671f710-dc7b-4cfa-bfe7-2252f0b6c25a)
![image](https://github.com/blai30/Lotus-Theme/assets/124537276/328e3135-d9f2-4373-8779-ab6499b0f341)

The Build (hammer) and More Actions (three dots) icons are barely visible when using Lotus Light on the New UI. It makes it hard to see what I'm pressing.

**Comment by blai30** [comment]
Fixed in 1.11 https://github.com/blai30/Lotus-Theme/releases/tag/1.11

**Comment by keriefie** [comment]
> Fixed in 1.11 https://github.com/blai30/Lotus-Theme/releases/tag/1.11

I'm curious why it was chosen that the toolbar should be dark for the light theme? Themes like Catppuchin have a light toolbar but with dark icons.

![image](https://github.com/blai30/Lotus-Theme/assets/124537276/50e0c808-b812-4810-b625-39451caf5998)

Additionally, this makes the text in the run config unreadable

![image](https://github.com/blai30/Lotus-Theme/assets/124537276/77b56dea-71f6-484c-a4c2-634befc004de)

**Comment by blai30** [comment]
I did not realize it was possible to change icon colors for the New UI, I just looked at the default light theme and saw it had a dark toolbar so went with that. The environments I test in does not have the hammer button, and the three dots button is slightly visible in yellow. 
I will submit another update that reverts the toolbar to light color and attempt to change the icon colors. If this does not help, feel free to clone the repo and help contribute as I do not plan to maintain this theme any longer.
